babel.config.js

```diff
@@ -1,5 +1,3 @@
-const babelFeaturesPlugin = require('@lwc/features/src/babel-plugin');
```

Comment:
I guess this is no longer needed after we moved all our tests to integration-karma